common/changes/@typespec/compiler/move-list-decorator_2023-06-20-11-35.json

```
@@ -0,0 +1,20 @@
+{
+  "changes": [
+    {
+      "packageName": "@typespec/compiler",
+      "comment": "Deprecate `@list` decorator in favor of `@listsResource` in `@typespec/rest`",
+      "type": "none"
+    },
+    {
+      "packageName": "@typespec/compiler",
+      "comment": "Deprecate `isListOperation` function in favor of `isListOperation` in `@typespec/rest`",
+      "type": "none"
+    },
+    {
+      "packageName": "@typespec/compiler",
+      "comment": "Deprecate `getListOperationType` function",
```

```suggestion
      "comment": "**Deprecate** `getListOperationType` function",
```
just so it is more visible in the changelog